gridstatus/caiso/caiso_constants.py

```diff
@@ -109,6 +118,48 @@
         },
         "params": {"market_run_id": "DAM"},
     },
+    "hasp_renewable_forecast_hourly": {
+        "query": {
+            "path": "SingleZip",
+            "resultformat": 6,
+            "queryname": "SLD_REN_FCST",
+            "version": 1,
+        },
+        "params": {
+            "market_run_id": "HASP",
+        },
+        "meta": {
+            "max_query_frequency": "1d",
+        },
+    },
+    "rtd_renewable_forecast": {
+        "query": {
+            "path": "SingleZip",
+            "resultformat": 6,
+            "queryname": "SLD_REN_FCST",
+            "version": 1,
+        },
+        "params": {
+            "market_run_id": "RTD",
+        },
+        "meta": {
+            "max_query_frequency": "1h",
```

Comment:
I'm not sure how we determine what this should be. I changed it to "1d", things worked fine, and it cut down the number of API requests by a factor of 24.

Reply:
My guess is there's a maximum number of rows the API will return at a time.